octodns_powerdns/__init__.py

```diff
@@ -364,7 +384,7 @@ def populate(self, zone, target=False, lenient=False):
             exists = True
             for rrset in resp.json()['rrsets']:
                 _type = rrset['type']
-                if _type == 'SOA':
+                if _type == 'SOA' or _type == 'DS':
```

Comment:
This should actually be checking `self.SUPPORTS` rather than explicitly listing out types. I'm guessing when it was originally implemented that wasn't yet the practice and no one has run across having `DS` records with PowerDNS.

```suggestion
                if _type in self.SUPPORTS:
```

Reply:
I agree that it is a better fix:
`if not _type in self.SUPPORTS:`
When running the test though, 2 tests fail and I get 2 warnings:
```
WARNING  PowerDnsProvider[test]:base.py:172 URLFWD records not supported for urlfwd.unit.tests.; omitting record
WARNING  PowerDnsProvider[test]:base.py:172 DNAME records not supported for dname.unit.tests.; omitting record
```
Those 2 type of records are unsupported, but in `tests/config/unit.tests.yaml`. I can easily fix the test (it is just lowering the number), but the warnings are emitted from octodns directly. Not sure what is the best way to handle that

Reply:
Made the change before I saw the comment.